webapp/template_functions.py

```diff
@@ -49,3 +48,19 @@ def format_number(number: int):
     with a comma between every thousands
     """
     return '{:,}'.format(number)
+
+
+def generate_slug(path):
```

Comment:
It's a bit confusing that this function is here, because it seems all the other functions from here are meant to be used in templates, while this one is only used in the preprocessor.

Reply:
@bartaz because it's a var in the templates I wanted to keep all that bizniz together. @tbille what do you think?

Reply:
@bartaz After discussion with @tbille we decided to rename the file and add comments to differentiate the different types of functions related to templates.